packages/fxa-payments-server/src/styles/tailwind.css

```diff
@@ -95,6 +95,10 @@
   @apply rounded-t-lg;
 }
 
+.page-title-container {
```

Comment:
Let's use this in `SubscriptionTitle` component (I think it's safe to remove `border-y-auto` in the className in that component as well, I noticed it wasn't `yarn build`-ing correctly with it)